model/riscv_types.sail

```diff
@@ -6,7 +6,7 @@
 /*  in the prover_snapshots directory (which include copies of their                     */
 /*  licences), is subject to the BSD two-clause licence below.                           */
 /*                                                                                       */
-/*  Copyright (c) 2017-2021                                                              */
+/*  Copyright (c) 2017-2022                                                              */
```

Comment:
2022 or 2023?

Reply:
The change has been lingering on our tree since 2022.
I'll update to 2023 when merging.